src/clojure/clj_yaml/core.clj

```diff
@@ -114,7 +114,8 @@
 
   clojure.lang.Keyword
   (encode [data]
-    (name data))
+    ;; using clojure.core/name would drop the namespace
+    (subs (str data) 1))
```

Comment:
I checked how cheshire does this and it seems to use the same trick:

https://github.com/dakrone/cheshire/blob/9b4bf0d940a8ac5b798636a1298062311105a5e6/src/cheshire/generate.clj#L68

Just fyi :)

Reply:
_/me pats himself on the shoulder_